doc/sphinx/src/models.rst

```diff
@@ -3,3 +3,409 @@
+
+.. math::
+
+    E_S(\rho) = E_0 + \int\limits_{\rho_0}^{\rho}
+      \frac{P_S(\bar{\rho})}{\bar{\rho^2}}~\mathrm{d}\bar{\rho}
+
+.. math::
+
+    T_S(\rho)  = T_0
+      \begin{cases}
+        \left(\frac{\rho}{\rho_0} \right)^{\Gamma_0}  & \rho < \rho_0 \\
+        \exp \left( -Zy \right) \left(\frac{\rho}{\rho_0} \right)^{\Gamma_0 + Z}
+                                                      & \rho >= \rho_0
+      \end{cases}
+
+where :math:`A`, :math:`B`, :math:`C`, :math:`y`, and :math:`Z` are all
+user-settable parameters and again quantities with a subcript of :math:`0`
+refer to the reference state. The variable :math:`\bar{\rho}` is simply an
+integration variable. The parameter :math:`C` is especially useful for ensuring
+that the high-pressure portion of the shock Hugoniot does not cross that of the
+products.
+
+The settable parameters are the dimensionless parameters listed above as well as
+the pressure, density, temperature, energy, Gruneisen parameter, and constant
+volume specific heat capacity at the reference state.
+
+
+Davis Products EOS
+'''''''''''''''''''
+
+The Davis products EOS is created from the reference isentrope passing through
+the CJ state of the high explosive along with a constant heat capacity. The
+constant heat capacity leads to the energy being a simple funciton of the
+temperature deviation from the reference isentrope such that
+
+.. math::
+    
+    E(\rho, T) = E_S(\rho) + C_{V,0} (T - T_S(\rho)).
+
+The Gruneisen parameter is given by
+
+.. math::
+
+    \Gamma(\rho) = k - 1 + (1-b) F(\rho)
+
+where :math:`b` is a user-settable dimensionless parameter and :math:`F(\rho)`
+is given by
+
+.. math::
+
+    F(\rho) = \frac{2a (\rho V_{\mathrm{C}})^n}{(\rho V_{\mathrm{C}})^{-n}
+      + (\rho V_{\mathrm{C}})^n}.
+
+Here the calibration parameters :math:`a` and :math:`n` are dimensionless while
+:math:`V_{\mathrm{C}}` is given in units of specific volume.
+
+Finally, the pressure, energy, and temperature along the isentrope are given by
+
+.. math::
+    
+    P_S(\rho) = P_{\mathrm{C}} G(\rho) \frac{k - 1 + F(\rho)}{k - 1 + a}
+
+.. math::
+
+    E_S(\rho) = E_{\mathrm{C}} G(\rho) \frac{1}{\rho V_{\mathrm{C}}}
+
+.. math::
+
+    T_S(\rho) = T_{\mathrm{C}} G(\rho) \frac{1}{(\rho V_{\mathrm{C}})^{ba + 1}}
+
+where
+
+.. math::
+
+    G(\rho) = \frac{
+      \left( \frac{1}{2}(\rho V_{\mathrm{C}})^{-n} 
+        + \frac{1}{2}(\rho V_{\mathrm{C}})^n \right)^{a/n}}
+      {(\rho V_{\mathrm{C}})^{-(k+a)}}
+
+and
+
+.. math::
+
+    E_{\mathrm{C}} = \frac{P_{\mathrm{C}} V_{\mathrm{C}}}{k - 1 + a}.
+
+Here, there are four dimensionless parameters that are settable by the user,
+:math:`a`, :math:`b`:, :math:`k`, and :math:`n`, while :math:`P_\mathrm{C}`,
+:math:`E_\mathrm{C}`, :math:`V_\mathrm{C}` and :math:`T_\mathrm{C}` are tuning
+parameters with units related to their non-subscripted counterparts.
+
+
+Spiner EOS
+````````````
+
+Stellar Collapse EOS
+````````````````````
+
+EOSPAC EOS
+````````````
```

Comment:
@annapiegra do you want to take a stab at this in the same style as above? I can also do it. (Either way we can submit in a different PR. I'd like to get @jhp-lanl 's contribution in. :)

Reply:
That sounds good. I just wanted to break things up and get _something_ in on this section to lay the groundwork.